rotkehlchen/tests/unit/globaldb/test_correctness.py

```diff
@@ -0,0 +1,25 @@
+from eth_utils.address import to_checksum_address
+
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+
+
+def test_checksummed_values(globaldb: GlobalDBHandler):
+    """Test that addresses and identifiers have checksummed addresses"""
+    cursor = globaldb.conn.cursor()
+    cursor.execute(
+        'SELECT evm_tokens.identifier, address, name, symbol FROM evm_tokens JOIN '
+        'common_asset_details ON evm_tokens.identifier=common_asset_details.identifier '
+        'JOIN assets ON evm_tokens.identifier=assets.identifier',
+    )
+    identifiers = set()
+    for identifier, address, name, symbol in cursor:
+        checksummed_address = to_checksum_address(address)
+        assert checksummed_address == address
+        assert checksummed_address in identifier
+        assert name != identifier
+        assert symbol != identifier
+        identifiers.add(identifier)
+
+    cursor.execute('SELECT asset FROM multiasset_mappings')
+    assert {x[0] for x in cursor if not x.startswith('eip155')}.issubset(identifiers)
```

Comment:
should have run this locally and not just apply my suggestion which I just did from gituh interface. Probaly needs x[0].startswith ?